crypto/err/err.c

```diff
@@ -694,13 +705,35 @@ ERR_STATE *ERR_get_state(void)
     return state;
 }
 
+void *err_shelve_state(void)
```

Comment:
Please document new functions.

Reply:
They are more or less meant as internal functions.
similar to err_free_strings_int, which is not documented either.

Reply:
Or do you mean add comments to the new functions?

Reply:
Yes, add comments. The documentation requirement exists for all functions, but only the public functions are in pod files.

Reply:
And it can be something very simple